Vagrantfile

```diff
@@ -0,0 +1,20 @@
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/zesty64"
+  config.vm.network "private_network", type: "dhcp"
+  config.vm.synced_folder ".", "/src/mustard"
+  config.vm.synced_folder "../content", "/src/content"
```

Comment:
Thanks for the review.

I think it needs to be both, but your comment made me realise that the directory needs to be created, not just shared and cloned. I've pushed another version